### PR TITLE
Add plot_timeline function, test_case, and dependencies

### PR DESCRIPTION
**Pull request by huan-ds**
It is the first draft, need to add more test cases

### CODE REVIEW

File: tests/test_pytweet.py [hunk 1]
```diff
@@ -1,5 +1,55 @@
...
+    None
+        The test should pass and no asserts should be displayed.
+    """
+    # Calling helper function to create data
+    data = helper_create_data()
+    
+    # Test the Exception is correctly raised when the type of `df`
+    # argument is wrong.
+    with pytest.raises(Exception) as e:
+        assert plot_timeline('', 'date')
```
Comment (mmyz88) [inline tests/test_pytweet.py]:
```suggestion
        plot_timeline('', 'date')
```

assert doesn't really do anything here. Also might wanna consider indent the next line to fall under `with`

File: tests/test_pytweet.py [hunk 2]
```diff
@@ -1,5 +1,55 @@
...
+    assert str(e.value) == "The value of the argument 'df' " \
+                           "must be type of dataframe."
+    
+    with pytest.raises(Exception) as e:
+        assert plot_timeline(data, 123)
```
Comment (mmyz88) [inline tests/test_pytweet.py]:
```suggestion
       plot_timeline(data, 123)
```

**Review by mmyz88 — APPROVED** [review]

File: tests/test_pytweet.py (hunk 1 above)
Comment (huan-ds) [reply tests/test_pytweet.py]: Thanks, changed!

Comment (huan-ds) [reply tests/test_pytweet.py]: Changed!

**Review by mmyz88 — APPROVED** [review]
good work

**Pull request merged**